### DISCUSSION TITLE
ClassifyRKD function

### DISCUSSION THREAD

**Discussion by MACTCD68** [PARADISE-AAV/rkd_package#23] [Ideas]
@Selbosh and @Conorisco 

The ClassifyRKD function is develop at the moment to use 1 algorithm. I show an example in the RMD to use 2 times the function with the output of the first algorithm as input of the second one.
Does the user wish to use several algorithm in the future ?
If yes, how we implement this ?

**Comment by Selbosh** [comment]
Add a pipeline wrapper function that runs all the functions (while those functions are still exposed, internally or externally). Then each algorithm can be a different wrapper function (based on a different order or combination of inner functions) or even passed as parameters to the wrapper.
